.github/workflows/reusable_proxy_lib.yml

```diff
@@ -77,6 +77,16 @@ jobs:
         working-directory: ${{env.BUILD_DIR}}
         run: UMF_PROXY="page.disposition=shared-shm" LD_PRELOAD=./lib/libumf_proxy.so /usr/bin/date
 
+      # TODO enable the provider_file_memory_ipc test when the IPC tests with the proxy library are fixed
+      # see the issue: https://github.com/oneapi-src/unified-memory-framework/issues/864
+      - name: Run "ctest --output-on-failure" with proxy library and size.threshold=128
+        working-directory: ${{env.BUILD_DIR}}
+        run: >
+          UMF_PROXY="size.threshold=128"
+          UMF_LOG="level:debug;flush:debug;output:stderr;pid:yes"
+          LD_PRELOAD=./lib/libumf_proxy.so
+          ctest --output-on-failure -E provider_file_memory_ipc
```

Comment:
we should have a test where you use umfPoolByPtr() to check allocs smaller than threshold were not registered in the tracker

Reply:
This is this test.
Done

Reply:
I will add a test for that

Reply:
@bratpiorka tests added
Done

Reply:
@bratpiorka ping